Feature/organise flake actions dependencies

julesve opened the pull request:
* Run tests from github actions
* Remove different requirements files in favor of pip files
* Remove old travis file
* Cleanup README.md
* Introduce flake8 ruleset

mgvd commented:
Please exclude 'migrations' in flake8 config to prevent  warnings on auto generated files
Even without 'migrations' flake8 report 70+ problems, which must be solved in future update

In README.MD line 13, use the new repository location

I get an error when running pipenv install --dev. "ERROR: Couldn't install package: uwsgi
 Package installation failed..."
Good be related to my machine, had issues earlier with virt env.

julesve commented:
> Please exclude 'migrations' in flake8 config to prevent warnings on auto generated files Even without 'migrations' flake8 report 70+ problems, which must be solved in future update
> 
> In README.MD line 13, use the new repository location
> 
> I get an error when running pipenv install --dev. "ERROR: Couldn't install package: uwsgi Package installation failed..." Good be related to my machine, had issues earlier with virt env.

Added migrations and settings to be excluded in .flake8 file. Also updated the readme with the new link. Regarding your problem with uwsgi, I deleted the pipenv en reinstalled it succesfully. Might be related to your python version (and/or windows).
